index.js

```diff
@@ -49,37 +49,40 @@ class AddressFinder extends React.Component {
     const { chosenAddress, results, error, searchTerm } = this.state;
     return (
       <div>
-        <div>Address finder:</div>
+        <label htmlFor="search">Address finder:</label>
         <div className="input_box">
           <input
+            id="search"
+            name="search"
             className="input"
             value={searchTerm}
             onChange={this.handleChange}
           />
           {searchTerm.length !== 0 && (
-            <div
+            <button
               className="clear_button"
               onClick={() => this.clearSearchValue()}
             >
-              <img src="clear.svg" />
-            </div>
+              <img src="clear.svg" alt="Clear address finder" />
```

Comment:
the `alt` text here provides a meaningful alternative to the image. This would be read out by screen readers. This is a thorough guide to writing good alt text: http://webaim.org/techniques/alttext/